src/metabase/events/view_log.clj

```diff
@@ -89,35 +87,16 @@
 (derive ::dashboard-read :metabase/event)
 (derive :event/dashboard-read ::dashboard-read)
 
-(defn- readable-dashcard?
-  "Returns true if the dashcard's card was readable by the current user, and false otherwise. Unreadable cards are
-  replaced with maps containing just the card's ID, so we can check for this to determine whether the card was readable"
-  [dashcard]
-  (let [card (:card dashcard)]
-    (not= (set (keys card)) #{:id})))
-
 (m/defmethod events/publish-event! ::dashboard-read
-  "Handle processing for the dashboard read event. Logs the dashboard view as well as card views for each card on the
-  dashboard."
-  [topic {:keys [object user-id] :as event}]
+  "Handle processing for the dashboard read event. Logs the dashboard view. Card views are logged separately."
+  [topic {:keys [object-id user-id] :as event}]
   (span/with-span!
     {:name "view-log-dashboard-read"
      :topic topic
      :user-id user-id}
     (try
-      (let [dashcards (filter :card_id (:dashcards object)) ;; filter out link/text cards wtih no card_id
-            user-id   (or user-id api/*current-user-id*)
-            views     (map (fn [dashcard]
-                             {:model      "card"
-                              :model_id   (u/id (:card_id dashcard))
-                              :user_id    user-id
-                              :has_access (readable-dashcard? dashcard)
-                              :context    "dashboard"})
```

Comment:
Previously `view_log` entries for card views from dashboard views would have `context=dashboard`. Similarly if the card view was from [a question](https://github.com/metabase/metabase/pull/43102/files#diff-1397b5fa941ecc1fba4d421b65e5cc9a6d81f158d017e3ba25020bf707fb72abL55) it would have `context=question`.